src/RelayServer.ts

```diff
@@ -709,7 +712,7 @@ export class RelayServer extends EventEmitter {
       this.managerAddress,
       this.workerAddress
     );
-    await this.registrationManager.init();
+    await this.registrationManager.init(initialBlockToScan);
```

Comment:
I may be confused but why are we using the `initialBlockToScan` instead of `_lastScannedBlock`.